build.gradle

```diff
@@ -1252,6 +1255,18 @@ if (!project.getGradle().startParameter.isOffline() && !Boolean.getBoolean('DISA
     }
 }
 
+// If you want to add more cases to this task, implement them as arguments if total amount to print gets too large
+tasks.register('faq') {
+    group = 'GTNH Buildscript'
+    description = 'Prints frequently asked questions about building a project'
+
+    doLast {
+        print("If your build fails to fetch dependencies, they might have been deleted and replaced by newer " +
+            "versions. In this case, check out the sites from which they are pulled in " +
+            "build.gradle:buildscript.repositories. See if the versions you try to fetch are still there.")
```

Comment:
This is wrong, the buildscript repositories are for gradle plugin metadata. The actual repositories for dependencies come from the `repositories {}` block lower down and the repositories.gradle file which can be different in every project

Reply:
Thank you! I'll address that.